### PR TITLE
fix(react@15): Address unsupported attributes warnings

### PR DESCRIPTION
**Pull request by nLight**
This is a dcos-ui part of addressing react@15 wrong html attributes warnings.
`reactjs-components` part: https://github.com/mesosphere/reactjs-components/pull/407

They can totally go separately.

### CODE REVIEW

**Comment by nLight** [comment]
Alfred please retest this one

**Comment by nLight** [comment]
Alfred please retest this one

File: src/js/components/TimeAgo.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ class TimeAgo extends React.Component {
       <time
         title={DateUtil.msToUTCDate(time)}
         dateTime={DateUtil.msToUTCDate(time)}
-        {...Util.omit(this.props, ["prefix", "suppressSuffix", "time"])}
+        {...Util.omit(this.props, Object.keys(TimeAgo.propTypes))}
```
Comment (ahoskins) [inline src/js/components/TimeAgo.js]: Isn't this kind of strange removing all the `propTypes` from the read-only `props`? If this always returns `{}` why even have this line?

File: src/js/components/Image.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ class Image extends React.Component {
       <img
         className={classes}
         onError={this.onImageError}
-        {...Util.omit(props, ["className"])}
+        {...Util.omit(props, ["className", "fallbackSrc"])}
```
Comment (ahoskins) [inline src/js/components/Image.js]: It seems like we would only ever want to pass the `src` prop, so why not do `src={props.src}` directly?

File: src/js/components/TimeAgo.js (hunk 1 above)
Comment (nLight) [inline src/js/components/TimeAgo.js]: This is a good question, I was just following a pattern, applying it to all the cases. I'll research how we're using the `TimeAgo` component and if we don't pass anything except what is in the propTypes I'd say we remove the line 👍

File: src/js/components/Image.js (hunk 2 above)
Comment (nLight) [inline src/js/components/Image.js]: I'll research if we're not missing anything 👍

**Comment by nLight** [comment]
@ahoskins looks like we can just remove it in both cases. Thanks for pointing this out!

**Comment by nLight** [comment]
retest this please

**Comment by nLight** [comment]
Alfred please retest this one

**Pull request merged**